homeassistant/components/freebox/alarm_control_panel.py

```diff
@@ -0,0 +1,170 @@
+"""Support for Freebox alarms."""
+import logging
+from typing import Any
+
+from homeassistant.components.alarm_control_panel import (
+    AlarmControlPanelEntity,
+    AlarmControlPanelEntityFeature,
+)
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import (
+    STATE_ALARM_ARMED_AWAY,
+    STATE_ALARM_ARMED_NIGHT,
+    STATE_ALARM_ARMING,
+    STATE_ALARM_DISARMED,
+    STATE_ALARM_TRIGGERED,
+    STATE_IDLE,
+)
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+
+from .const import DOMAIN, FreeboxHomeCategory
+from .home_base import FreeboxHomeEntity
+from .router import FreeboxRouter
+
+FREEBOX_TO_STATUS = {
+    "alarm1_arming": STATE_ALARM_ARMING,
+    "alarm2_arming": STATE_ALARM_ARMING,
+    "alarm1_armed": STATE_ALARM_ARMED_AWAY,
+    "alarm2_armed": STATE_ALARM_ARMED_NIGHT,
+    "alarm1_alert_timer": STATE_ALARM_TRIGGERED,
+    "alarm2_alert_timer": STATE_ALARM_TRIGGERED,
+    "alert": STATE_ALARM_TRIGGERED,
+}
+
+
+_LOGGER = logging.getLogger(__name__)
+
+
+async def async_setup_entry(
+    hass: HomeAssistant, entry: ConfigEntry, async_add_entities: AddEntitiesCallback
+) -> None:
+    """Set up alarm panel."""
+    router: FreeboxRouter = hass.data[DOMAIN][entry.unique_id]
+
+    alarm_entities: list[AlarmControlPanelEntity] = []
+
+    for node in router.home_devices.values():
+        if node["category"] == FreeboxHomeCategory.ALARM:
+            alarm_entities.append(FreeboxAlarm(hass, router, node))
+
+    if alarm_entities:
+        async_add_entities(alarm_entities, True)
+
+
+class FreeboxAlarm(FreeboxHomeEntity, AlarmControlPanelEntity):
+    """Representation of a Freebox alarm."""
+
+    def __init__(
+        self, hass: HomeAssistant, router: FreeboxRouter, node: dict[str, Any]
+    ) -> None:
+        """Initialize an alarm."""
+        super().__init__(hass, router, node)
+
+        self._attr_extra_state_attributes = {}
+        self._state: str | Any
```

Comment:
```suggestion
```

Do not create your custom attributes and properties if they exists from parent class: use ones from `FreeboxHomeEntity`, `AlarmControlPanelEntity` or its parent `Entity`